plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/navigator/database/StatisticsNavigatorNodeRenderer.java

```diff
@@ -601,4 +619,10 @@ private static boolean isHorizontalScrollbarEnabled(Tree tree) {
         }
         return horizontalBar.isVisible();
     }
+
+    private static boolean isNodeActionsEnabled() {
+        DBPPlatform platform = DBWorkbench.getPlatform();
+        DBPPreferenceStore preferenceStore = platform.getPreferenceStore();
+        return preferenceStore.getBoolean(NavigatorPreferences.NAVIGATOR_SHOW_NODE_ACTIONS);
```

Comment:
A bit mouthful, I'd get rid of both local variables.

Reply:
I thought about it for a second. and I've settled on the current variant because, without these variables, the method doesn't fit on small screens.